lib/fs.js

```diff
@@ -63,6 +63,18 @@ const isWindows = process.platform === 'win32';
 const DEBUG = process.env.NODE_DEBUG && /fs/.test(process.env.NODE_DEBUG);
 const errnoException = util._errnoException;
 
+let truncateWarn = true;
+
+function doFlaggedDeprecation() {
```

Comment:
Just a suggestion, but I’d include `truncate` in the function name here because it’s specific to `fs.truncate(Sync)`

Reply:
Very good suggestion, I will keep this in mind in my future works. Thanks